hybridcloud/backends/azure_postgresqlflexible.py

```diff
@@ -208,21 +210,51 @@ def compare():
         except ResourceNotFoundError:
             applied_allowed_extenions = []
         
+        should_restart = False
```

Comment:
Maybe document why this is done :) something along the lines of "Since changes to the server parameters require changes, keep track of whether a restart is required"